test/CapabilityDelegator_tests.cdc

```diff
@@ -1,140 +1,211 @@
 import Test
 import "test_helpers.cdc"
+import "CapabilityDelegator"
+import "NonFungibleToken"
+import "ExampleNFT"
 
-pub let adminAccount = blockchain.createAccount()
-pub let creator = blockchain.createAccount()
+access(all) let admin = Test.getAccount(0x0000000000000007)
+access(all) let creator = Test.createAccount()
 
-pub let flowtyThumbnail = "https://storage.googleapis.com/flowty-images/flowty-logo.jpeg"
+access(all) let flowtyThumbnail = "https://storage.googleapis.com/flowty-images/flowty-logo.jpeg"
 
 // BEGIN SECTION - Test Cases
 
-pub fun testSetupDelegator() {
+access(all)
+fun testSetupDelegator() {
     setupDelegator(creator)
 
-    let typ = CompositeType("A.01cf0e2f2f715450.CapabilityDelegator.DelegatorCreated")!
-    let events = blockchain.eventsOfType(typ)
+    let typ = Type<CapabilityDelegator.DelegatorCreated>()
```

Comment:
So much nicer!

Reply:
Ah yes, good idea :+1: 
Updated in https://github.com/onflow/hybrid-custody/pull/161/commits/5946bbf1c1d30ece1b36926948687069969aac89